changelogs/fragments/6212-pipx-include-injected.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

This seems more like a feature to me.
```suggestion
minor_changes:
```